### PR TITLE
Aura scripts

### PR DESCRIPTION
**Pull request by nvitucci**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-aura-214.surge.sh

File: README.adoc [hunk 1]
```diff
@@ -10,4 +10,63 @@ npm start
...
+=== Setup
+
+1. Copy `aura_api.env.example` to `aura_api.env` and update it to use actual API access credentials (client ID and client secret).
+2. Create a Python virtual environment: `python -m venv venv`.
+3. Activate the virtual environment: `source venv/bin/activate`.
+4. Install the requirements: `pip install -r scripts/requirements.txt`.
+
+=== Running the script
+
+1. Activate the virtual environment (if not already active): `source venv/bin/activate`.
```
Comment (MarkWoulfeNeo4j) [inline README.adoc:30]:
```suggestion
1. Activate the virtual environment (if not already active): `source venv/bin/activate` (`venv\Scripts\activate` on Windows).
```

File: README.adoc [hunk 2]
```diff
@@ -10,4 +10,63 @@ npm start
...
+This script can be used to extract runnable code (Python or Cypher statements) from a doc HTML page. This means that the docset has to be built first, for example by running `npm run build`.
+
+NOTE: The script requires Python 3. 
+
+NOTE: AuraDS credentials must be available in an `aura.env` file.
+
+=== Setup
+
+1. Create a Python virtual environment: `python -m venv venv`.
+2. Activate the virtual environment: `source venv/bin/activate`.
```
Comment (MarkWoulfeNeo4j) [inline README.adoc:47]:
```suggestion
2. Activate the virtual environment: `source venv/bin/activate` (`venv\Scripts\activate` on Windows).
```

File: README.adoc [hunk 3]
```diff
@@ -10,4 +10,63 @@ npm start
...
+== Replace partials in an AsciiDoc file
+
+Some applications cannot use Asciidoc files that use partials. The `replace_partials.py` script can be used to preprocess an Asciidoc file and replace the `include::partial` lines with the actual content of the referenced partials. 
+
+NOTE: The script requires Python 3.
+
+=== Setup
+
+1. Create a Python virtual environment: `python -m venv venv`.
+2. Activate the virtual environment: `source venv/bin/activate`.
```
Comment (MarkWoulfeNeo4j) [inline README.adoc:66]:
```suggestion
2. Activate the virtual environment: `source venv/bin/activate` (`venv\Scripts\activate` on Windows).
```

File: README.adoc [hunk 4]
```diff
@@ -10,4 +10,63 @@ npm start
...
+3. Install the requirements: `pip install -r scripts/requirements.txt`.
+
+=== Running the script
+
+1. Activate the virtual environment (if not already active): `source venv/bin/activate`.
```
Comment (MarkWoulfeNeo4j) [inline README.adoc:52]:
```suggestion
1. Activate the virtual environment (if not already active): `source venv/bin/activate` (`venv\Scripts\activate` on Windows).
```

File: README.adoc [hunk 5]
```diff
@@ -10,4 +10,63 @@ npm start
...
+NOTE: The script requires Python 3.
+
+=== Setup
+
+1. Create a Python virtual environment: `python -m venv venv`.
+2. Activate the virtual environment: `source venv/bin/activate`.
+3. Install the requirements: `pip install -r scripts/requirements.txt`.
+
+=== Running the script
+1. Activate the virtual environment (if not already active): `source venv/bin/activate`.
```
Comment (MarkWoulfeNeo4j) [inline README.adoc:70]:
```suggestion
1. Activate the virtual environment (if not already active): `source venv/bin/activate` (`venv\Scripts\activate` on Windows).
```

**Review by MarkWoulfeNeo4j — APPROVED** [review]
Added windows commands but looks good apart from that.

Note: I only tested the instance creation and deletion as I have no use for the other scripts, i will trust your judgement on those.

**Pull request merged**